pkgs/fujiwara/tfstate-lookup/registry.yaml

```diff
@@ -0,0 +1,6 @@
+packages:
+  - type: github_release
+    repo_owner: fujiwara
+    repo_name: tfstate-lookup
+    description: Lookup resource attributes in tfstate
+    asset: "tfstate-lookup_{{trimV .Version}}_{{.OS}}_{{.Arch}}.tar.gz"
```

Comment:
```suggestion
    asset: "tfstate-lookup_{{trimV .Version}}_{{.OS}}_{{.Arch}}.tar.gz"
    supported_if: GOOS != "windows"
```

Currently, we're working on the Windows Support.
This tool doesn't support Windows, so please set `supported_if` to ignore Windows.
https://github.com/aquaproj/aqua/issues/850

And please run `bash generate-registry.sh`.